ceph-releases/kraken/ubuntu/16.04/demo/entrypoint.sh

```diff
@@ -215,6 +215,12 @@ function bootstrap_rgw {
     mkdir -p /var/lib/ceph/radosgw/${RGW_NAME}
     ceph ${CEPH_OPTS} auth get-or-create client.radosgw.gateway osd 'allow rwx' mon 'allow rw' -o /var/lib/ceph/radosgw/${RGW_NAME}/keyring
     chown -R ceph. /var/lib/ceph/radosgw/${RGW_NAME}
+
+    #configure rgw dns name
+    cat <<ENDHERE >>/etc/ceph/${CLUSTER}.conf
```

Comment:
In that case, do you want to move it out of 
```if [ ! -e /var/lib/ceph/radosgw/${RGW_NAME}/keyring ]```   condition? 

I though that was enough in case of container restart, but can change that.